AK/Bitmap.h

```diff
@@ -163,69 +157,151 @@ class Bitmap {
         return {};
     }
 
-    Optional<size_t> find_longest_range_of_unset_bits(size_t max_length, size_t& found_range_size) const
+    /**
+     * the function is supposed to find the next range of unset bits 
+     * starting from the @from value. 
+     * @from: the postition from which the search starts. The var
+     *        will be changed and new value is the offset of the
+     *        found block
+     * @min_length: minimum size of the range which will be returned
+     * @max_length: shows when to stop adding unset bits to the range.
+     *            This is used for optimisation, since the range of
+     *            unset bits can be long, and we don't need the whole
+     *            size, so we can stop when we've reached @max_length.
+     * It's recomended to use @min_length and @max_length, since they
+     * give a boost.
+     */
```

Comment:
Please use C++ `//` comments.
The function is not *supposed* to find the next range, it *will* find it :)